src/shadowbox/server/manager_service.spec.ts

```diff
@@ -110,10 +112,87 @@ describe('ShadowsocksManagerService', () => {
     });
   });
 
+  describe('changeHostname', () => {
```

Comment:
nit: this could be more descriptive "Changes the hostname for access keys"

Reply:
Will resolve once we all agree on whether or not this affects all keys or new keys

Reply:
Done